nbs/animation.ipynb

```diff
@@ -197,6 +198,23 @@
         "[args_d.update(a.param.values()) for a in arg_objs]\n",
         "args=AnimationArgs(**args_d)\n",
         "\n",
+        "\n",
+        "# load override settings if provided\n",
+        "if len(override_settings_path):\n",
+        "    if not os.path.exists(override_settings_path):\n",
+        "        raise ValueError(f\"Override settings file not found: {override_settings_path}\")\n",
+        "    with open(override_settings_path, 'r') as f:\n",
+        "        overrides = json.load(f)\n",
+        "        args = vars(args)\n",
+        "        for k in args.keys():\n",
+        "            if k in overrides:\n",
+        "                args[k] = overrides[k]\n",
+        "        args = SimpleNamespace(**args)\n",
+        "        animation_prompts = overrides.get('animation_prompts', animation_prompts)\n",
+        "        animation_prompts = {int(k): v for k, v in animation_prompts.items()}\n",
+        "        negative_prompt = overrides.get('negative_prompt', negative_prompt)\n",
```

Comment:
to do: negative prompts are just prompts with a negative sign on the weight. we shouldn't treat them special in our api/sdk tooling (although it's fine to treat them special in an interface layer over our tooling like a notebook)

Reply:
Agreed yeah, it's been broken out because expectation was we'd test it this way for NSFW filter false positive mitigations with animation.